sentry-ruby/lib/sentry/hub.rb

```diff
@@ -67,6 +67,23 @@ def pop_scope
       @stack.pop
     end
 
+    def get_transaction
+      current_scope.span
+    end
+
+    def start_span(**options)
+      if span = current_scope.span
+        span.start_child(**options)
+      else
+        Span.new(**options)
+      end
+    end
```

Comment:
Shouldn't exists anymore, spans are only created as child of transactions or other spans.